lib/browser.js

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+import historyPolyfill from 'html5-history-api';
```

Comment:
I'd recommend users to overwrite `window.history` with the polyfill if the object is `undefined`. The polyfill you found, [html5-history-api](https://github.com/devote/HTML5-History-API), is spec compliant, so this wouldn't cause any issues. This is how most libraries handle polyfills now AFAIK.